parsely-amp-tracking.php

```diff
@@ -0,0 +1,25 @@
+add_action( 'pre_amp_render_post', 'parsely_amp_add_actions' );
+function parsely_add_amp_actions() {
+    add_filter( 'amp_post_template_analytics', 'parsely_amp_add_analytics' );
```

Comment:
This will become `add_filter('amp_post_template_analytics', array($this, 'parsely_add_amp_analytics'));`, again since it's in the `Parsely` class.

Note again the consitency for function naming--let's name this callback `parsely_add_amp_analytics`